### ISSUE TITLE
VC-Tools are not available

### ISSUE THREAD

**Issue by FooDeas** [FooDeas/raspberrypi-ua-netinst#6]
The Raspberry Pi tools like `vcgencmd`, `raspistill` and similar are not available without removing kernel.

**Comment by FooDeas** [comment]
The issue was a pinning of some packages from raspbian.org, which is not needed anymore.
Fix for existing systems:

```
sudo rm /etc/apt/preferences.d/01vcgencmd.pref

sudo sh -c "echo \"Package: *
Pin: origin archive.raspberrypi.org
Pin-Priority: 600
\" > /etc/apt/preferences.d/archive_raspberrypi_org.pref"

sudo apt-get install device-tree-compiler libraspberrypi-bin libraspberrypi0
```

**Comment by thijstriemstra** [comment]
Thanks!

**Comment by FooDeas** [comment]
Fixed with a574957c908b79adbca0502b286a07818576bd9d

**Comment by bassemolufda** [comment]
Thanks.
